crates/server/primitives/src/admin.rs

```diff
@@ -272,6 +272,25 @@ impl<T> ListAliasesResponse<T> {
     }
 }
 
+pub trait Report {
+    fn report(&self) -> String;
+}
+
+impl Report for ListAliasesResponse<PublicKey> {
+    fn report(&self) -> String {
+        use std::fmt::Write;
+        let mut output = String::new();
+
+        writeln!(output, "{c1:44} | {c2}", c1 = "Identity", c2 = "Alias").unwrap();
+
+        for AliasRecord { alias, value } in &self.data {
+            writeln!(output, "{c1:44} | {c2}", c1 = value, c2 = alias).unwrap();
+        }
+
+        output
+    }
+}
+
```

Comment:
this should be in meroctl, and it should use the same table format we use everywhere else